cypress/e2e/functional/tile_tests/data_card_tool_spec.js

```diff
@@ -137,5 +137,23 @@ context('Data Card Tool Tile', () => {
     dc.getLinkGraphModalTileMenu().select('Data Card Collection 1');
     dc.getLinkGraphModalLinkButton().should("contain", "Link").click();
     xyplot.getXAxisLabel().should("contain", "habitat");
+
+    cy.log("Copy card functionality");
+   // Select a card
+   dc.getTile().click();
+   // Click the duplicate card button
+   dc.getDuplicateCardButton().should('not.be.disabled').click();
+   // Number of cards should increase by 1
+   // New card has focus/is in view.
+   dc.getCardNofTotalListing().contains("Card 3 of 3");
+   // New cards can be in the middle of the deck
+   dc.getPreviousCardButton().click();
+   dc.getDuplicateCardButton().should('not.be.disabled').click();
+   dc.getCardNofTotalListing().contains("Card 3 of 4");
+   // Data can be changed/is not linked to original card
+   dc.getAttrValue().eq(0).dblclick().type("{backspace}river{enter}");
+   dc.getAttrValue().eq(1).dblclick().type("{backspace}{backspace}{backspace}{backspace}{backspace}rhinocerotter{enter}");
```

Comment:
Uisng clear() might be considered an improvement over the `{backspace}` in typing. So the above two lines could be:

```suggestion
   dc.getAttrValue().eq(0).dblclick().clear().type("river{enter}");
   dc.getAttrValue().eq(1).dblclick().clear().type("rhinocerotter{enter}");
```